3.2/root/usr/share/container-scripts/mongodb/mongodb.conf.template

```diff
@@ -6,13 +6,13 @@ port = ${CONTAINER_PORT}
 # Default: /var/lib/mongodb/data
 dbpath = ${MONGODB_DATADIR}
 
-# Disable data file preallocation. Default: true
+# Disable data file preallocation.
 # only for mounted data directory from older MongoDB server
-noprealloc = ${MONGODB_NOPREALLOC}
+noprealloc = true
```

Comment:
Well, if we remove the user's ability to set that via env var, shouldn't we just leave the configuration to use the default values?

IIUC, this setting only applies to users migrating from an older version to 3.2 with existing data (in the old storage format).